deps/rabbitmq_mqtt/test/auth_SUITE.erl

```diff
@@ -451,6 +471,34 @@ user_credentials_auth(Config) ->
         fun(Conf) -> connect_user(<<"non-existing-vhost:guest">>, <<"guest">>, Conf) end,
         Config).
 
+client_id_from_cert_san_dns(Config) ->    
+    setup_rabbit_auth_backend_mqtt_mock(Config),
+
+    ExpectedClientId = <<"rabbit_client_id">>, % Found in the client's certificate as SAN type CLIENT_ID 
+    MqttClientId = <<"ignored_client_id">>,
```

Comment:
see my comment above, defining two different MQTT client IDs for the same MQTT session is confusing in both client and server